rd_ui/app/views/dashboard.html

```diff
@@ -58,7 +58,7 @@ <h3 class="panel-title">
                 </div>
             </div>
 
-            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
+            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
```

Comment:
In what way it wasn't the same?
On Sep 11, 2015 4:21 PM, "jmvasquez" <notifications@github.com> wrote:

> In rd_ui/app/views/dashboard.html
> <https://github.com/EverythingMe/redash/pull/551#discussion_r39269822>:
>
> > @@ -58,7 +58,7 @@ <h3 class="panel-title">
> >                  </div>
> >              </div>
> >
> > -            <div class="panel panel-default rd-widget-textbox" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
> > +            <div class="panel panel-default rd-widget-textbox" ng-style="widget.disableWidget" ng-if="type=='textbox'" ng-mouseenter="showControls = true" ng-mouseleave="showControls = false">
>
> When I tried to do it, the widget was not present (instead hidden) in the
> page, so the functionality was not the same. That was my previous
> experience.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/EverythingMe/redash/pull/551/files#r39269822>.
>